<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,8">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_superior">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>42</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color: rgb(89, 214, 50);
}

QPushButton{
background-color: #000000ff;
border-radius: 20px;
}

QPushButton:hover{
background-color: #afffab;
border-radius: 20px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="btn_menu">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>31</width>
            <height>24</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../img/menu.png</normaloff>../img/menu.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_contenido">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frame_control">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>0</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color: rgb(0, 206, 151);
}


QPushButton{
background-color: rgb(61, 61, 61);
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
color: rgb(255, 255, 255);
font: 77 10pt &quot;Arial Black&quot;;
}

QPushButton:hover{
background-color: White;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
color: rgb(0, 0, 0);
font: 77 10pt &quot;Arial Black&quot;;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Basa de Datos</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Registrar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Actualizar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Eliminar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>Asco</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_paginas">
            <property name="styleSheet">
             <string notr="true">QFrame {
background-color: rgb(61, 61, 61);
}

QLabel{
font: 87 12pt&quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(0, 206, 151);
border:0px solid #14C8DC
}

QLineEdit{
border: 0px;
color: rgb(255, 255, 255);
border-bottom:2px solid rgb(61,61,61);
font: 75 12pt &quot;Times New Roman&quot;;
}

QPushButton{
background-color: rgb(61, 61, 61);
border-radius: 15px;
color: rgb(255, 255, 255);
font: 77 10pt&quot;Arial Black&quot;;
}

QPushButton:hover{
background-color: rgb(0, 206, 151);
border-radius: 15px;
color: rgb(0, 0, 0);
font: 77 10pt&quot;Arial Black&quot;;
}

QTableWldget {
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 206, 151);
font-size: 12pt;
color: #000000;
}

QHeaderView::section {
background-color: rgb(0, 206, 151);
border: 1px solid rgb(0, 0, 0);
font-size: 12pt;
}

QTableWidget QTableCornerButton::section {
background-color: rgb(0, 0, 0);
border: 1px solid rgb(0, 206, 151);
}
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item>
              <widget class="QStackedWidget" name="stackedWidget">
               <widget class="QWidget" name="page_datos">
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Servicio al Cliente</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTableWidget" name="tabla_servicio">
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <column>
                    <property name="text">
                     <string>No. Orden</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Cliente</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Motivo</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Fecha</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Estado</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="btn_refrescar">
                     <property name="minimumSize">
                      <size>
                       <width>120</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Refrescar</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_actualizar"/>
               <widget class="QWidget" name="page_eliminar"/>
               <widget class="QWidget" name="page_ajustes"/>
               <widget class="QWidget" name="page_registrar">
                <widget class="QPushButton" name="pushButton_6">
                 <property name="geometry">
                  <rect>
                   <x>260</x>
                   <y>330</y>
                   <width>75</width>
                   <height>24</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>PushButton</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_7">
                 <property name="geometry">
                  <rect>
                   <x>130</x>
                   <y>20</y>
                   <width>49</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_8">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>330</y>
                   <width>49</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
                <widget class="QWidget" name="">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>50</y>
                   <width>239</width>
                   <height>239</height>
                  </rect>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <property name="spacing">
                     <number>30</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_6">
                      <property name="text">
                       <string>TextLabel</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <property name="spacing">
                     <number>30</number>
                    </property>
                    <item>
                     <widget class="QLineEdit" name="lineEdit"/>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_2"/>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_3"/>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_4"/>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_5"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
